src/views/ocr/components/OcrResultPanel.vue

```
@@ -0,0 +1,244 @@
+          </a-collapse>
+        </div>
+      </a-spin>
+    </a-card>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { Message } from '@arco-design/web-vue'
+import { ocrInfer } from '@/apis/ocr'
+import type { OcrPageResult } from '@/apis/ocr'
+import OcrResultContent from './OcrResultContent.vue'
```

ESLint

[error] `./OcrResultContent.vue` import should occur before import of `@/apis/ocr`

(import/order)